README.md

```diff
@@ -8,24 +9,28 @@
 ## Features
 
 A customizable banner to highlight to the user that the website is in demo, dev
-or staging state. 
+or staging state.
 
-It is configurable through the `settings.stagingBanner.demoIdentifiers` 
+It is configurable through the `settings.stagingBanner.demoIdentifiers`
 (list of identifiers such as e.g. `true`, `dev`, `demo` discoverable from reading
 the value of `settings.stagingBanner.envRuntimeVariable` which by default reads
 the value of `RAZZLE_DEMO_SITE` environment variable), the parent element
 where to add the banner (`settings.stagingBanner.parentNodeSelector`) and through
 the addition of extra css classes (`settings.stagingBanner.classes`).
 
+if you want to customize banner header and content, you can use `RAZZLE_BANNER_HEADER` and `RAZZLE_BANNER_MESSAGE` env vars respectively.
```

Comment:
Use this phrasing instead as it fixes some grammatical errors:
If you want to customize the banner header and content, you can use `RAZZLE_BANNER_HEADER` and `RAZZLE_BANNER_MESSAGE` env vars, respectively.